### PR TITLE
Add a line text on homepage

### PR DESCRIPTION
**Pull request by windsonsea**
Add what Merbridge can support on the home page.

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="docs/getting-started">
 		Get Started <i class="fab fa-github ml-2 "></i>
 	</a>
-	<p class="lead mt-5">Let's explore Merbridge</p>
+	<p class="lead mt-5">Well supports Istio, Linkerd2, and Kuma.</p>
```
Comment (hanxiaop) [inline content/en/_index.html]:
```suggestion
	<p class="lead mt-5">Currently supported Istio, Linkerd2, and Kuma.</p>
```

Comment (windsonsea) [reply content/en/_index.html]: `currrently` is a bit contradictory with `-ed`, how about this change:
```
Istio, Linkerd2, and Kuma are supported currently.
```

Comment (hanxiaop) [reply content/en/_index.html]: `Currently supported sth` and `sth is currently supported` are basically the same thing. Either of them are ok to me

Comment (windsonsea) [reply content/en/_index.html]: Well, modified. Shorter is better here.

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**